DESCRIPTION

```diff
@@ -9,14 +9,11 @@ Authors@R: c(
     person("Jeffrey", "Dickinson", role = "aut"),
     person("Ross", "Farrugia", role = "aut"),
     person("Fanny", "Gautier", role = "aut"),
-    person("G", "Gayatri", role = "aut"),
-    person("Asha", "Chakma", role = "aut"),
     person("Dinakar", "Kulkarni", role = "aut"),
```

Comment:
Should Dinakar get moved to acknowledgements?

Reply:
He still supports us so I'd keep